src/sentry/api/event_search.py

```diff
@@ -662,3 +662,150 @@ def get_snuba_query_args(query=None, params=None):
             kwargs['has_boolean_terms'] = True
             kwargs['conditions'].append(convert_search_boolean_to_snuba_query(term))
     return kwargs
+
+
+FIELD_ALIASES = {
+    'issue_title': {
+        'aggregations': [['anyHeavy', 'title', 'issue_title']],
+    },
+    'last_seen': {
+        'aggregations': [['max', 'timestamp', 'last_seen']],
+    },
+    'latest_event': {
+        'fields': [
+            ['argMax', ['id', 'timestamp'], 'latest_event'],
+        ],
+    },
+    'project': {
+        'fields': ['project.id'],
+    },
+    'user': {
+        'fields': ['user.id', 'user.name', 'user.username', 'user.email', 'user.ip'],
+    }
+    # TODO(mark) Add rpm alias.
+}
+
+VALID_AGGREGATES = {
+    'count_unique': {
+        'snuba_name': 'uniq',
+        'fields': '*',
+    },
+    'count': {
+        'snuba_name': 'count',
+        'fields': '*'
+    },
+    'avg': {
+        'snuba_name': 'avg',
+        'fields': ['issue.id', 'duration'],
+    },
+    'min': {
+        'snuba_name': 'min',
+        'fields': ['timestamp', 'issue.id', 'duration'],
```

Comment:
min, max and avg `issue.id` seems very odd to me. Although they are ascending over time i assume that's not a detail we'd want to call out in any way.

Reply:
I agree. I'll take issue.id out for now.